src/executables/util_cs.c

```diff
@@ -3720,6 +3733,10 @@ start_ddl_proxy_client (const char *program_name, DDL_CLIENT_ARGUMENT * args)
       au_enable (save);
     }
 
+  locator_all_flush ();
```

Comment:
Does this affect current ha_replication? If review takes long, this should be quickly merged.

Reply:
I made a test with this flush removed, and the copy process works without it.
I may have introduce it as a fix during a phase when the copy apply did not work, but issue was probably a different one.
I will try to investigate if is necessary...

Reply:
The call to locator_all_flush is not necessary here and I will remove it.
We already have in do_execute_statement:
```suggestion
  else if (db_is_ddl_proxy_client ())
    {
      /* We need to flush in case of ddl proxy. */
      if (err >= 0)
	{
	  err = locator_all_flush ();
	}
    }
```

Also, some paths indicate that after each class class schema change, there is a flush to server:
dbt_finish_class -> sm_finish_class -> update_class -> allocate_disk_structures -> locator_flush_class.  (I confirmed this using a test on replication copy code)